### ISSUE TITLE
Builder fails to load on certain layout

### ISSUE THREAD

**Issue by dm-chechik** [elegantthemes/Divi-Beta#746]
### Problem:

The issue comes from the customer.
http://www.studiojoe.us/
Neither builder (backend and frontend) do not handle the homepage layout correctly.
I've tried to load the layout into my test environment and nothing changed.
Maybe it related to the large image in Fullwidth Header module.

**Link To Download Layout JSON File**:
https://www.dropbox.com/s/63ouz9ac8yjta5r/Home-page-layout.zip?dl=0

**Comment by SlavaET** [comment]
I found 2 unrelated errors with the provided layout:
- It was an issue with the content inside the Pricing table. It was handled incorrectly in VB and lead to js error. I improved handling of the content inside modules with child items ( like Pricing Tables, Sliders, etc ) in my PR.
- Af for the BB - I was able to load this layout in BB, but it was an issue in the FW header because of wrong icon code and it lead to the js error as well, so it was impossible to edit Fullwidth Header in BB. Fixed this error in my PR.
